src/axom/mir/docs/sphinx/mir_algorithms.rst

```diff
@@ -7,11 +7,44 @@
 MIR Algorithms
 ******************************************************
 
-The MIR component contains MIR algorithms that will take a Blueprint mesh as input,
+The MIR component contains MIR algorithms that take a Blueprint mesh as input,
 perform MIR on it, and output a new Blueprint mesh with the reconstructed output.
 A Blueprint mesh is contained in a ``conduit::Node`` and it follows the `Blueprint protocol <https://llnl-conduit.readthedocs.io/en/latest/blueprint_mesh.html>`_,
 which means the node contains specific items that describe the mesh coordinates, topology, fields, and materials.
 
+The MIR algorithms provide a uniform interface that takes an input Blueprint mesh
+with a material set *(matset)* that defines how mesh zones are divided into materials.
+The MIR algorithms all output a new Blueprint mesh that contains new coordinates, topology,
+fields, and materials.
+
+Axom's MIR algorithms are data parallel and can run on the CPU and the GPU.
+Axom's implementation supports 2D/3D zones from structured or unstructured topologies
+made of Finite Element Zoo elements *(e.g. triangles, quadrilaterals, tetrahedra, pyramids,
+wedges, hexahedra, or topologically-compatible mixtures)*. The MIR logic for Equi-Z is
+encapsulated in ``axom::mir::EquizAlgorithm``, which is a class that is templated on view objects.
+View objects help provide an interface between the Blueprint data and the MIR algorithm.
+At a minimum, an execution space and three views are required to instantiate a MIR algorithm.
+The execution space determines which compute backend will
+be used to execute the algorithm. The Blueprint data must exist in a compatible
+memory space for the execution space. The views are: *CoordsetView*, *TopologyView*, and
+*MaterialView*. The *CoordsetView* template argument lets the algorithm access the mesh's
+coordset using concrete data types and supports queries that return points. The
+*TopologyView* provides a set of operations that can be performed on meshes, mainly a
+method for retrieving individual zones that can be used in device kernels.
+The *MaterialView* provides an interface for querying matsets.
```

Comment:
Since we have an ELVIRA section, should parts of this be refactored into an Equi-Z section?